wordpress/wordpress/Dockerfile_LOCAL

```diff
@@ -0,0 +1,99 @@
+# =============================================================================
+# wordpress:5-php7.3-apache
+#
+# WordPress 5.*, Apache 2.4, PHP 7.3
+# Modification: Rsyslog/Fail2Ban setup
+#
+# =============================================================================
+FROM wordpress:5-php7.3-apache
+LABEL maintainer="Jacob Thompson <jacob.thompson@wiwasolvet.ca>"
+
+RUN apt-get update &&  \
+    apt-get install -y \
+    nano \
+    vim \
+    rsyslog \
+    apt-utils \
+    zlib1g-dev \
+    brotli \
+    libtidy-dev \
+    apache2-dev \
+    git
+#    && rm -r /var/lib/apt/lists/*
+
+# Adds necessary rsyslog format for Fail2Ban regex
+RUN sed -i -e 's|\$ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat|#\$ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat|g' /etc/rsyslog.conf && \
+    echo '$template CustomFormat,"%timestamp:::date-year%-%timestamp:::date-month%-%timestamp:::date-day% %timestamp:::date-hour%:%timestamp:::date-minute%:%timestamp:::date-second% %HOSTNAME% %syslogtag%%msg%\\n' >> /etc/rsyslog.conf && \
+    echo '$ActionFileDefaultTemplate CustomFormat' >> /etc/rsyslog.conf
+
+
+# Adds setting for Apache to ensure Fail2Ban correctly uses proxy IP on production server setup. i.e. ban the client IP not the proxy IP.
+RUN a2enmod remoteip && \
+    touch /etc/apache2/conf-enabled/remoteip.conf && \
+    printf "%s\n" "RemoteIPHeader X-Forwarded-For" \
+           "RemoteIPTrustedProxy " + PROXY_IP > /etc/apache2/conf-enabled/remoteip.conf
+
+# service apache2 restart
+
+# Sets timezone which normally uses dpkg-reconfigure tzdata (2 for America, 65 for Halifax)
+ENV TZ=America/Halifax
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN docker-php-ext-install tidy
+
+# Enable memcache and memcached
+RUN mkdir -p /usr/src/php/ext
```

Comment:
Compare with prod Dockerfile, and disable memcache etc for time being until we move to docker swarm etc. load balancers?